.gitignore

```diff
@@ -13,6 +13,7 @@ wordclock_config/wordclock_config.cfg
 
 # ignore swap-files
 *.swp
+.vscode
```

Comment:
Use `.vscode*` to include vscode/tags, etc.

Reply:
resolved